resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                notOpenStatus:
+                  $ref: '#/components/examples/notOpenStatusUnauthorized'
+      deprecated: false
+      security:
+        - user-restricted:
+            - read:lisa
+  /lifetime-isa/manager/{lisaManagerReferenceNumber}/investors:
+    post:
+      tags:
+        - LISA Investors
+      summary: Create a LISA investor
+      description: |
+        Report a new LISA investor to HMRC to generate an investor ID. If the investor already exists, you will get their reference number.
+        
+        ### Test data
+
+        <table style="font-family:Roboto, sans-serif; font-size:14px;">
+            <col width="25%">
+            <col width="35%">
+            <col width="40%">
+            <thead>
+                <tr>
+                    <th>Scenario</th>
+                    <th>Request Payload</th>
+                    <th>Response</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td><p>Request with a valid payload and LISA Manager reference number</p> <p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorNINO": "AA123456A",
+          "firstName": "First Name",
+          "lastName": "Last Name",
+          "dateOfBirth": "1985-03-25"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">201 (Created)</code></p>
+        <pre class="code--block">
+        {
+          "status": 201,
+          "success": true,
+          "data": {
+            "message": "Investor created",
+            "investorId": "9876543210"
+          }
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request with a valid payload and an invalid LISA Manager reference number</p> <p class="code--block">lisaManagerReferenceNumber: 123456</p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorNINO": "AA123456A",
+          "firstName": "First Name",
+          "lastName": "Last Name",
+          "dateOfBirth": "1985-03-25"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+        {
+          "code": "BAD_REQUEST",
+          "message": "Enter lisaManagerReferenceNumber in the correct format, like Z1234"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing invalid and/or missing data</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks